apps/settings/js/downloads/downloads_list.js

```diff
@@ -457,5 +465,4 @@
 }(window));
 
 // startup
-navigator.mozL10n.once(DownloadsList.init.bind(DownloadsList));
```

Comment:
does it means we can do `mozL10n.formatValue` directly without waiting for mozL10n ready?